docs/command/mongocli-atlas-liveMigrations.txt

```diff
@@ -0,0 +1,61 @@
+.. _mongocli-atlas-liveMigrations:
+
+=============================
+mongocli atlas liveMigrations
+=============================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Manage a Live Migration to Atlas for your organization.
+
+Manage a Live Migration to Atlas for your organization.
```

Comment:
I was going to comment on the command but here's more obvious, I think we can just remove long since it just duplicates short

Reply:
Good spot, very obvious now. Removing the long